.circleci/config.yml

```diff
@@ -13,32 +13,60 @@ commands:
     description: "Prepare tooling, setup path for environment"
     steps:
       - run:
-          name: Prepare codebase
+          name: Prepare tooling
           command: |
+            # Prepare some variables to use in commands.
+            ~/project/.circleci/prepare-environment-variables.sh
+            # Install Git.
             apt update && apt install git -y
+            # Install Mariadb Client for using mysql command like mysql, drush sql-cli ...
+            apt install mariadb-client -y
+            # Install Platformsh CLI to interact with Platformsh.
+            docker-php-ext-install pcntl
+            docker-php-ext-install posix
+            curl -sS https://platform.sh/cli/installer | php
+            echo "export PATH=$PATH:$HOME/.platformsh/bin" >> $BASH_ENV
+            source ~/.bashrc
+            # Skip host key checking for Platformsh
+            mkdir -p ~/.ssh
+            echo -e "Host *.platform.sh\n  StrictHostKeyChecking no" >> ~/.ssh/config
+            # Make some project commands are able to call from terminal like drush, drupal, phpcs, phpunit...
             echo "export PATH=$PATH:$HOME/project/vendor/bin" >> $BASH_ENV
-            composer install -d /root/project
+      - run:
+          name: Prepare codebase
+          command: |
+            composer install -d ~/project
             rm -rf /var/www/html
-            ln -s /root/project/web/ /var/www/html
+            ln -s ~/project/web/ /var/www/html
```

Comment:
What is the reason for changing this? Is it sometimes not /root?

Reply:
Yes. I think it sometimes is not root. Using `~` is more flexible.

Reply:
No worries - it doesn't matter if it works :)